### DISCUSSION TITLE
Clearer indication of referenced line number in PR review comments

### DISCUSSION THREAD

**Discussion by adigitoleo** [community/community#139497] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I suggest adding a clearer visual indication of which line number is being referenced in a PR review comment. Because a few lines of surrounding context are shown in the PR discussion tab, it is not immediately clear which line is being referenced, especially for new users who may not be aware that the comment always refers to the immediately preceding line (I think). For example, I recently got a review comment that asked to remove a blank line, but in a different location than I instinctively thought.

Because a highlight color, on top of the already existing red/green diff highlights, would probably introduce too much visual clutter, I instead suggest that the indicator take the form of a dot or symbol in the "sign column" (in Vim terminology) i.e. the visual column immediately preceding the line numbers.

I think this is a small change that would result in a nice QOL boost and better productivity.

**Comment by abdessalems** [comment]
Hi there,

I appreciate your suggestion for enhancing the PR review experience on GitHub! A clearer visual indication for referenced line numbers would indeed help both new and experienced users avoid confusion during reviews.

Your idea of using a dot or symbol in the "sign column" to indicate the line being referenced is particularly interesting. It could provide a subtle yet effective way to draw attention to the specific line without adding visual clutter from additional colors.

Implementing this small change could significantly improve the quality of life for users by making it easier to understand comments in context, thus increasing overall productivity. I recommend submitting your feedback through the GitHub feedback portal, as they often consider user suggestions for future updates.

Thanks for sharing your thoughts!

**Reply by adigitoleo** [reply]
This response seems... generated? In any case, I think "the GitHub feedback portal" is exactly where we having this discussion now.
